shell/key-bindings.bash

```diff
@@ -79,7 +74,7 @@ else
 fi
 
 # CTRL-R - Paste the selected command from history into the command line
-bind -m emacs-standard '"\C-r": " \C-e\C-u\C-y\ey\C-u`__fzf_history__`\e\C-e\er\e^"'
+bind -m emacs-standard '"\C-r": "\C-k\C-uHISTCMD=$HISTCMD\e\C-e __fzf_history__`\"\C-a\"`\e\C-e\C-e\er"'
```

Comment:
The reason we have `<space>` at the beginning of the sequence is to suppress beep/bell from the terminal. It beeps when `<ctrl-u>` is pressed on an empty command line. You can test it on tmux by enabling visual bell like so:

```sh
make docker

tmux set -g visual-bell on

# <CTRL-R>
# Visual bell rings
```

- ` `: Paste empty space
- `\C-e`: Move the cursor to the end
- `\C-u`: Delete the whole command line. We can be sure that the command line is not empty, so no beep.
- `\C-y\ey`: Remove the new entry (possibly just an empty space) in the "kill ring"

Reply:
:man_facepalming: Whoops! Thank you for explaining, now I get it and I've undone that change. Thanks!